pkg/api/instance/instance.go

```diff
@@ -83,7 +83,10 @@ func getVersion() string {
 
 // Derive the URL from the verrazzano URI by replacing the 'api' segment
 func deriveURL(s string) string {
-	return "https://" + s + "." + verrazzanoURI
+	if len(strings.TrimSpace(verrazzanoURI)) > 0 {
```

Comment:
i don't see how this does what the comment above the message says it does - am i missing something?

Reply:
this function previously existed, as did the comment. It is referring to the fact that it is "deriving" a full URL given a prefix i.e. from the prefix `s` and the previously set `verrazzanoURI` (which is really an env + dnsZone suffix), it derives the full URL, as in `return "https://" + s + "." + verrazzanoURI`

Reply:
ah, I see it says `api` in the comment - never mind, will make it more accurate

Reply:
done - clarified the comment